lib/shape/CollideCirclePolygone.js

```diff
@@ -150,4 +150,51 @@ function CollidePolygonCircle(manifold, polygonA, xfA, circleB, xfB) {
     manifold.points[0].id.cf.indexB = 0;
     manifold.points[0].id.cf.typeB = Manifold.e_vertex;
   }
-}
+};
+
+Contact.polygonCircleOverlap = function(polygonA, xfA, circleB, xfB) {
+  // Compute circle position in the frame of the polygon.
+  cLocal = Transform.mulVec2(xfB, circleB.m_p, cLocal);
+  cLocal = Transform.mulTVec2(xfA, cLocal, cLocal);
+
+  // Find the min separating edge.
+  var normalIndex = 0;
+  var separation = -Infinity;
+  var radius = polygonA.m_radius + circleB.m_radius;
+  var vertexCount = polygonA.m_count;
+  var vertices = polygonA.m_vertices;
+  var normals = polygonA.m_normals;
+
+  for (var i = 0; i < vertexCount; ++i) {
+    var s = Vec2.dot(normals[i], d.set(cLocal).sub(vertices[i]));
+
+    if (s > radius) {
+      // Early out.
+      return;
+    }
+
+    if (s > separation) {
+      separation = s;
```

Comment:
We are trying to find polygon edge which is most "turned" to the circle, this one will have the greatest separation. Not sure why comment was about "min separating edge", removed "min" to avoid confusion.